src/components/AssetInfo/TokenSideBarBeneficiary.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ReactElement } from "react";
 import css from "./TokenSideBar.scss";
 import TokenSideBarField from "./TokenSideBarField";
-import { TOKEN_ACTION_TYPES } from "./util";
+import { TOKEN_ACTION_TYPES } from "./../../utils/TokenActionUtil";
```

Comment:
why moved to global util folder?  I think its utility for token interaction only.